src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNodeTests.cs

```diff
@@ -2802,6 +2802,44 @@ class C
             });
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/19613")]
+        public void TestRemove_KeepUnbalancedDirectives_Indented()
+        {
+            var inputText = """
+                class C
+                {
+                    // before
+                    #region Fred
+                    // more before
+                    void M()
+                    {
+                    } // after
+                    #endregion
+                }
+                """;
+
+            var expectedText = """
+                class C
+                {
+
```

Comment:
unfortunate blank line.  i'm considering chnaging that in this PR to not happen either.  LMK if you'd prefer that.

Reply:
i looked into changing this and it was a wash.  Undesirable lines like this were kept, while desirable lines (like those between methods) were removed.  So i'm keeping this for now.